gulpfile.js

```diff
@@ -211,6 +211,27 @@ function clean_cache() {
     return del(['./cache/**'], { force: true });
 };
 
+function dist_filter_libraries() {
+    // Only grab what we need
+    var distSources = [
+        './node_modules/bluebird/js/browser/bluebird.min.js',
+        './node_modules/i18next/i18next.min.js',
```

Comment:
I don't know if this is the way to go. 
We must maintain here the list of sources. I prefer to grab the complete library but I suppose that I will be the only one with this. Let's wait others opinions.

Reply:
The problem with shipping the complete library is the amount of extra files. 


Version|files in node_modules
-------|------
10.2.0| 70
10.3.0| 81
10.4.0| 110
10.5.0| 105
10.6.0| 404 so far

I think only jbox changed the name of the sources.

Reply:
I wouldn't worry too much about extra files - thanks to running in NW.js, our distribution files are in the 70 MB range anyway, a couple of (hundred) js source files will not make a significant difference.